test/contrib/snowflake_test.py

```diff
@@ -0,0 +1,373 @@
+import os
+import sys
+
+import mock
+from moto import mock_s3
+
+import luigi
+import luigi.contrib.snowflake
+import luigi.notifications
+
+from helpers import unittest, with_config
+
+from nose.plugins.attrib import attr
+
+if (3, 4, 0) <= sys.version_info[:3] < (3, 4, 3):
+    # spulec/moto#308
+    mock_s3 = unittest.skip('moto mock doesn\'t work with python3.4')  # NOQA
+
+
+BUCKET = 'bucket'
+KEY = 'key'
+
+
+class DummyTarget(luigi.contrib.snowflake.SnowflakeTarget):
+    pass
+
+
+class DummyS3CopyToTableBase(luigi.contrib.snowflake.S3CopyToTable):
+    host = 'dummy_host'
+    database = 'dummy_database'
+    user = 'dummy_user'
+    password = 'dummy_password'
+    table = luigi.Parameter(default='dummy_table')
+    columns = luigi.TupleParameter(
+        default=(
+            ('some_text', 'varchar(255)'),
+            ('some_int', 'int'),
+        )
+    )
+
+    copy_options = ''
+
+    def s3_load_path(self):
+        return 's3://%s/%s' % (BUCKET, KEY)
+
+
+class DummyS3CopyToTableWithKeys(DummyS3CopyToTableBase):
+    aws_access_key_id = 'property_key'
+    aws_secret_access_key = 'property_secret'
+
+
+@attr('snowflake', 'contrib')
```

Comment:
not sure this needs to be added to multiple test groups

Reply:
Do you have a preference between setting it to `contrib` vs `snowflake`?

Reply:
🤔 

This is a good question bc it starts to break my original line of thinking. I had included an `AWS` test section, which this _could_ fit into. However, it's not exclusive to AWS.

Let's go with `contrib` for now